src/lugo4py/rl/gym.py

```diff
@@ -61,24 +73,16 @@ async def helper_players(gameServerAddress):
         return self
 
 
-async def create_helper_players(gameServerAddress: str, training_bot_number, training_team_side):
-    tasks = []
+def create_helper_players(gameServerAddress: str):
+
+    # await newZombieHelperPlayer(Team.Side.HOME, 1, gameServerAddress)
     for i in range(1, 12):
-        if i == training_bot_number and training_team_side is not None:
-            if training_team_side == Team.Side.HOME:
-                tasks.append(newZombieHelperPlayer(
-                    Team.Side.AWAY, i, gameServerAddress))
-            elif training_team_side == Team.Side.AWAY:
-                tasks.append(newZombieHelperPlayer(
-                    Team.Side.HOME, i, gameServerAddress))
-            continue
-
-        tasks.append(newZombieHelperPlayer(
-            Team.Side.HOME, i, gameServerAddress))
-        tasks.append(newZombieHelperPlayer(
-            Team.Side.AWAY, i, gameServerAddress))
-
-    return await asyncio.gather(*tasks)
+        print(f'PLEAYR {i}\n')
+        asyncio.ensure_future(newZombieHelperPlayer(Team.Side.AWAY, i, gameServerAddress))
+        asyncio.ensure_future(newZombieHelperPlayer(Team.Side.HOME, i, gameServerAddress))
```

Comment:
@Ddarckwolf , isso nao ta funcionando.
Nao sei pq, mas a funcao newZombieHelperPlayer nunca eh executada.
Deixei um log pra ver se ela eh pelo menos chamada, mas nao eh.

Essa chamada taqui nao pode ser bloqueante, mas se colocar o `await` se torna bloqueante.

Ve se vc acha uma forma de executar e nao bloquear.

O termo que o pessoal usa eh `fire and forget`, tentei algumas solucoes sugeridas no Stackoverflow, mas nenuma deu certo.